package.json

```diff
@@ -110,7 +110,7 @@
     "electron-is-dev": "^1.1.0",
     "electron-rabbit": "^1.0.3",
     "electron-timber": "^0.5.1",
-    "electron-updater": "^4.2.2",
+    "electron-updater": "^4.3.9",
```

Comment:
Wasn't sure if we should pin this but decided against doing so since I don't think it's strictly tied to the `electron-builder` version